src/ext/extension.ts

```diff
@@ -164,7 +164,7 @@ export function activate(context: vscode.ExtensionContext) {
     context.subscriptions.push(vscode.debug.registerDebugConfigurationProvider('cc65-vice', provider));
 
     console.log('Running as ', runMode);
-    let factory: vscode.DebugAdapterDescriptorFactory;
+    let factory: CC65ViceDebugAdapterDescriptorFactory | InlineDebugAdapterFactory | DebugAdapterExecutableFactory;
     // Do I need this or will it be broken?
```

Comment:
The newer TypeScript complier complains below because 'vscode.DebugAdapterDescriptiorFactory' never contains a 'dispose()' method.

```
  Property 'dispose' is missing in type 'DebugAdapterDescriptorFactory' but required in type '{ dispose(): any; }'.
src/webviews/stats-webview-content.ts(351,75): error TS2349: This expression is not callable.
```